backoffice/src/Repositories/DroitRepository.cs

```diff
@@ -89,6 +91,21 @@ public async Task<Droit> GetDroitAsync(Guid id)
 
         return droit;
     }
+    
+    public async Task<Droit> GetDroitByReferenceAsync(string reference)
+    {
+        var droit = await Context.Droits
+                                .Include(d => d.WreckMaterials).ThenInclude(wm => wm.Images)
```

Comment:
You shouldn't need any of these, just the droit itself without any of the children. Triage is a flat field on the droit itself